### PR TITLE
Name change opentacos

### LINKED ISSUE #20: Change "open-tacos" to "climb database" or climbing route catalog
### Background
**open-tacos** is the code name for the [coding project](https://github.com/OpenBeta/open-tacos) that powers our main website https://openbeta.io

When the project first started, the website is also called "open-tacos" (old domain: tacos.openbeta.io).

### Proposal
To simplify our brand messaging, only use OpenBeta or the "climb database" or the climb route catalog. Retire "open tacos" in any public facing documentation.

When a coder wants to learn more contributing code, we can point them to [open-tacos](https://github.com/OpenBeta/open-tacos) repository on GitHub.  Other than that we should only use  OpenBeta to refer to our main project, the website, or the climb database or the climbing catalog.

### PR DESCRIPTION
**Pull request by clintonlunn**
fixes #20 

Removed references to OpenTacos.
Switched to using Yarn to match other projects and installation instructions.
Changed the folder name for improved clarity.
Updated all references to the previous name throughout the codebase.

so far looks good except for the search bar. I cannot seem to get this name to change. I'm not too familiar with docusaurus in general. Maybe it's being cached?

image

### CODE REVIEW

**Comment by actuallyyun** [comment]
Hey Clinton, thanks for making the changes. I think it has conflicts with dev branch now as I merged the updated overview and how to contribute file.

File: package-lock.json
Comment (actuallyyun) [inline package-lock.json:1]: Nice you removed it. I was confused by the two lock files. Since you added `yarn.lock`, I will not commit mine.

**Comment by musoke** [comment]
By default, search is provided by algolia and configured at https://github.com/OpenBeta/docs.openbeta.io/blob/04e1bca817c22e39eac1e62e958823790cb4ae47/docusaurus.config.js#L131-L151
The index won't update until they crawl the updated site (~1 per week?). 

https://docusaurus.io/docs/search

**Comment by vnugent** [comment]
The crawler runs automatically once per week, but I can start it manually.

**Comment by clintonlunn** [comment]
@vnugent I am trying to see why the vercel deploy failed, but when I try to visit the [link](https://openbeta-docs-ifzbpuv6v-openbeta-dev.vercel.app/) provided in the vercel message above, I get 404.

**Comment by vnugent** [comment]
@clintonlunn I've relaxed the log protection setting. Can you try again?  It's complaining about broken links

**Comment by clintonlunn** [comment]
> @clintonlunn I've relaxed the log protection setting. Can you try again?  It's complaining about broken links

I can see the logs now. Ty!

**Review by vnugent — APPROVED** [review]

**Pull request merged**